frontend/src/metabase/questions/containers/QuestionIndex.jsx

```diff
@@ -20,110 +20,129 @@ import { getAllCollections, getAllEntities } from "../selectors";
 import { getUserIsAdmin } from "metabase/selectors/user";
 
 import { replace, push } from "react-router-redux";
+import EmptyState from "metabase/components/EmptyState";
+
+export const CollectionEmptyState = () =>
+    <div className="flex align-center p2 mt4 bordered border-med border-brand rounded bg-grey-0 text-brand">
+        <Icon name="collection" size={32} className="mr2"/>
+        <div className="flex-full">
+            <h3>Create collections for your saved questions</h3>
+            <div className="mt1">
+                Collections help you organize your questions and allow you to decide who gets to see what.
+                {" "}
+                <a href="http://www.metabase.com/docs/latest/administration-guide/06-collections.html" target="_blank">
+                    Learn more
+                </a>
+            </div>
+        </div>
+        <Link to="/collections/create">
```

Comment:
I'm not sure, I have thought that the methods in `lib/urls` are primarily helpers for constructing more complicated urls.